### PR TITLE
Improve test coverage

### PR DESCRIPTION
**Pull request by andy-faunadb**
Should wait on #80 and probably #81

### CODE REVIEW

File: .gitignore [hunk 1]
```diff
@@ -9,5 +9,6 @@ dist
 docs/_build
 htmlcov/
 faunadb.egg-info
+htmlcov
```
Comment (ashfire908) [inline .gitignore]: Duplicate

File: README.rst [hunk 2]
```diff
@@ -81,6 +81,14 @@ Run ``pip install .[doc]`` to install the needed packages to generate the docs.
 Then run ``make doc``, then open ``docs/_build/html/index.html`` in a web browser.
 
 
+Coverage
```
Comment (ashfire908) [inline README.rst]: Duplicate

**Comment by ashfire908** [comment]
I'd actually like to ship this ahead of #81. We can wait till after that's fixed to reach 100% coverage across all versions.

**Comment by andy-faunadb** [comment]
OK, but #88 should get in first.

**Comment by ashfire908** [comment]
Please fix failing test

**Comment by ashfire908** [comment]
LGTM

**Pull request merged**